### PR TITLE
Add monitor app

### PR DESCRIPTION
**Pull request by djaciel**

### CODE REVIEW

File: packages/monitor/src/monitorHandler.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  prevTimestamp: string
+}
+
+async function sendTelegramMessage(message: string) {
+  if (!process.env.TELEGRAM_BOT_TOKEN || !process.env.TELEGRAM_CHAT_ID) {
+    log(chalk.bold.red('TELEGRAM_BOT_TOKEN and TELEGRAM_CHAT_ID must be provided'))
+    return
+  }
+
+  const bot = new TelegramBot(process.env.TELEGRAM_BOT_TOKEN, { polling: false })
```
Comment (arthurka-o) [inline packages/monitor/src/monitorHandler.ts:41]: `TelegramBot` will be initialized every time we send a message. Will it make sense to initialize `bot` variable in global space and use `||=` to initialize/assign?

File: packages/monitor/src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  '0xe060ea13b2e710cefc5124bb790db4823b0f602a': 'Ink',
+  '0xef898dbf4F4D75bdfbDd85F781A6C1BF8EDaF0AE': 'NodeMonkes',
+  '0x770E3A8afC5c01855b5bD8EB5b96b23bd7Af1e43': 'EvIndex',
+  '0xad90fFf9D159e18CEc2048Dd6881e29886e4899E': 'AiIndex',
+}
+
+export const executorAddresses = [
+  '0xAA26EBdCe83a5fC137355C3F1Ef28358cdBde693',
+  '0x043539595f87e879B53789486987B1be4b43441c',
+]
```
Comment (TomasCImach) [inline packages/monitor/src/constants.ts:16]: can you get this from the pk on the .env file?

File: packages/monitor/src/monitorHandler.ts (hunk 1 above)
Comment (djaciel) [reply packages/monitor/src/monitorHandler.ts]: done!

File: packages/monitor/src/constants.ts (hunk 2 above)
Comment (djaciel) [reply packages/monitor/src/constants.ts]: Done!, now i'm is getting from redis

**Review by TomasCImach — APPROVED** [review]

**Pull request merged**